packages/themes/src/themes/console/light.ts

```diff
@@ -192,9 +192,9 @@ export const lightTheme = {
             fillWeakDisabled: '#f3f3f4',
           },
           warning: {
-            fill: '#ffeccc',
```

Comment:
No need to update them in dark mode?

Reply:
@QuiiBz nope in dark mode they are okay :p 

<img width="78" alt="Screenshot 2023-07-21 at 11 42 25" src="https://github.com/scaleway/ultraviolet/assets/15812968/5d4db494-79bd-4a57-991b-51b48562a801">